test/acceptance/behave/components/ngsiv2/subscriptions/create_a_new_subscription/create_a_new_subscription_notification.feature

```diff
@@ -225,29 +268,56 @@ Feature: create new subscriptions (POST) using NGSI v2. "POST" - /v2/subscriptio
       | http://www.foo.bar./       |
       | http://.www.foo.bar./      |
```

Comment:
Why not add invalid ports as well?
E.g.:

    http://1.1.1.1:70000  # port is a short int, mas valid is lessa than 7000 (65535)
    http://1.1.1.1:abc    # port must be an integer